ci/run_lint.sh

```diff
@@ -12,7 +12,7 @@ rm -f "$END_MARKER"
 
 # Run all tests if no input argument is given
 if (($# == 0)); then
-  tests=('clint-full' 'lualint' 'pylint' 'shlint' 'check-single-includes')
+  tests=('lint_c_full' 'lint_lua' 'lint_py' 'lint_sh' 'check-single-includes')
```

Comment:
I don't think underscores are used in any of our other build targets, so let's stay consistent. Could use hyphens I guess, which is in some of our build targets, but that is annoying to type/complete.

```suggestion
  tests=('lintcfull' 'lintlua' 'lintpy' 'lintsh' 'check-single-includes')
```